code/modules/medical/computer/cloning.dm

```diff
@@ -16,6 +16,7 @@
 	var/datum/dna2/record/active_record = null
 	var/obj/item/weapon/disk/data/diskette = null //Mostly so the geneticist can steal everything.
 	var/loading = 0 // Nice loading text
+	var/available_species = list("human","tajaran","skrell","unathi","grey","plasmamen","vox")
```

Comment:
>de-hardcode

Fuck off goof. The list is limited for a reason. If it was "de-hardcoded" we'd have a longer list of races we don't want.